src/TypingPage/TypingPage.tsx

```diff
@@ -27,10 +28,19 @@ class TypingPage extends React.Component<TypingProps, TypingState> {
     constructor(props: TypingProps) {
         super(props);
 
+        let count = 0;
+        const insults = this.props.insults;
+
+        //count chars in insults
+        for(let i = 0; i < insults.length; i++) {
+            count += insults[i].length;
+        }
+
         this.state = {
             currentInsult: 0,
             typedText: "",
-            isFinished: false
+            isFinished: false,
+            numChars: count
```

Comment:
Is this used anywhere? Or was it part of the "double work" for calculating the typing speed?

Reply:
Good catch!